src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/ArrivalEntity.kt

```diff
@@ -0,0 +1,42 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity
+
+import java.time.LocalDate
+import java.util.UUID
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.OneToOne
+import javax.persistence.Table
+
+@Entity
+@Table(name = "arrival")
```

Comment:
I guess this is more of a convention/style thing but tend to pluralise table names.

Reply:
Ah hadn't realised the convention, since we've got some existing tables will leave it as-is for this PR then switch them all over with a migration in a follow-up PR.

Reply:
Unless this has gone live do you need a migration to rename them?

Reply:
I mean that since we already have other tables from previous PRs that are named in the singular rather than the plural (and have been deployed to dev) it would make sense to move the entire project from singular->plural in one PR

Reply:
Ah right sorry.

Reply:
This was addressed in https://github.com/ministryofjustice/approved-premises-api/pull/30